api/api/src/main/scala/uk/ac/wellcome/platform/api/models/SearchTemplates.scala

```diff
@@ -0,0 +1,22 @@
+package uk.ac.wellcome.platform.api.models
+
+import com.sksamuel.elastic4s.requests.searches.queries.{Query, QueryBuilderFn}
+import io.circe.generic.extras.semiauto.deriveConfiguredEncoder
+import io.circe.Encoder
+import uk.ac.wellcome.display.json.DisplayJsonUtil._
+
+case class SearchTemplates(templates: List[SearchTemplate])
+object SearchTemplates {
+  implicit def encoder: Encoder[SearchTemplates] =
+    deriveConfiguredEncoder
+}
```

Comment:
I’m surprised you need to declare encoders explicitly; we usually handle this by importing `JsonUtil._`.

Reply:
Personally I think it's better to derive explicitly, as otherwise we end up in a situation where the same encoder is automatically derived multiple times and really slow compilation times.

This should be an `implicit val` rather than `implicit def` by the looks of thing though.

Reply:
This also slows down compilation significantly.

Reply:
`valèd it up.